pkgs/tools/security/sequoia/default.nix

```diff
@@ -1,81 +1,38 @@
 { stdenv
 , fetchFromGitLab
-, fetchpatch
 , lib
 , darwin
-, git
 , nettle
 , nix-update-script
-, cargo
-, rustc
 , rustPlatform
 , pkg-config
 , openssl
 , sqlite
-, capnproto
-, ensureNewerSourcesForZipFilesHook
-, pythonSupport ? true
-, pythonPackages ? null
 }:
 
-assert pythonSupport -> pythonPackages != null;
-
 rustPlatform.buildRustPackage rec {
   pname = "sequoia";
```

Comment:
On a second thought, maybe this should be changed to? to `sequoia-sq` or simply `sq`? And maybe the attribute name should be changed as well? @minijackson what do you think?

Reply:
Looking at Repology, it seems like other distributions are adopting `sequoia-sq`:

- https://repology.org/project/sequoia-sq/versions
- https://repology.org/project/sq/versions
- https://repology.org/project/sequoia/versions

I think it would be nice to align with them

Reply:
Thanks for the links, I renamed it, along with an alias.